src/lib/crypto/kimchi_bindings/stubs/kimchi_bindings.ml

```diff
@@ -215,7 +215,8 @@ module Protocol = struct
         -> SRS.Fp.t
         -> bool
         -> t
-        = "caml_pasta_fp_plonk_index_create_bytecode" "caml_pasta_fp_plonk_index_create"
+        = "caml_pasta_fp_plonk_index_create_bytecode"
```

Comment:
ignorea

Reply:
Yeah... I wanted to make a patch to the formatter used in the mina codebase to ignore this newline change in kimchi_bindings and scalars, but I didn't spend time yet on it 👎🏻